### PR TITLE
Change default path to tc-print

### PR DESCRIPTION
**Pull request by paulbiss**
https://github.com/facebook/hhvm/commit/65fb8134f99af5200b094c0557b25e1d05a5195d

### CODE REVIEW

File: base/PerfOptions.php [hunk 1]
```diff
@@ -235,7 +235,7 @@ public function __construct(Vector<string> $argv) {
         $this->tcprint = $tcprint;
       } else if ($isFacebook) {
         $this->tcprint =
-          $fbcode . '/_bin/hphp/facebook/tools/tc-print/tc-print';
+          $fbcode . '/_bin/hphp/tools/tc-print/tc-print';
```
Comment (fredemmott) [inline base/PerfOptions.php]: We could infer this from $this->hhvm without gating by isFacebook now, couldn't we?

Comment (paulbiss) [inline base/PerfOptions.php]: The rest of that path is still facebook specific, our cmake build doesn't create an _bin directory. A flag could be added to search for things in the right place in a cmake build, I just haven't added it.

Comment (fredemmott) [inline base/PerfOptions.php]: can be done relative to $this->hhvm

**Comment by fredemmott** [comment]
bump

**Comment by fredemmott** [comment]
Stale.

**Pull request closed without merging**